scripts/release

```diff
@@ -325,7 +325,7 @@ build_and_push_package() {
   local staging_username=$7
   local staging_token=$8
 
-  PUSH_TO_REPO=$push \
+  PUSH_TO_REPO="no" \
```

Comment:
Double checking. Do we want "no" here?

Reply:
Yeah, this is intentional.  We want to just build the deb/rpm packages here since they'll now be signed/pushed separately.